CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to Open Liberty Guides
+
+Anyone can contribute to the Open Liberty guides and we welcome your contributions!
+
+There are multiple ways to contribute: report bugs, fix bugs, contribute code, improve upon documentation, etc. 
+
+## Raising Issues
+
+Please raise any bug reports on the [Open Liberty guide's repository's](https://github.com/OpenLiberty/guide-maven-intro/issues). For new topics, large updates to existing guides, or general suggestions and ideas, report your issue on the [Open Liberty common guides repository](https://github.com/OpenLiberty/guides-common/issues). Be sure to search the list to see if your issue has already been raised.
```

Comment:
Instead of using `https://github.com/OpenLiberty/guide-maven-intro/issues`, try use `../../issues`, so that this file can be used for all guides without update this url

Reply:
why cannot use `../.../issues`? 
I saw -> https://github.com/OpenLiberty/[guide-name]/issues, where [guide-name]
when I click on the link 404 error

Reply:
- Originally tried to make it in a way where reporting an issue on the repo is explicit.

Reply:
will be addressed by gkwan-ibm-patch-1 branch